nncf/pruning/utils.py

```diff
@@ -115,16 +116,15 @@ def traverse_function(node: NNCFNode, output, nncf_graph: NNCFGraph, type_check_
     return True, output
 
 
-def get_first_pruned_modules(target_model: NNCFNetwork, pruned_ops_types):
+def get_first_pruned_nodes(graph: NNCFGraph, pruned_ops_types):
```

Comment:
Since you touched this, please add the type hints for the remaining argument and the return value.

Reply:
Done